sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -251,6 +257,47 @@ def impl(self):
     gen_hpat_pandas_series_rolling_impl(arr_sum, float64))
 
 
+@sdc_overload_method(SeriesRollingType, 'apply')
+def hpat_pandas_series_rolling_apply(self, func, raw=None):
```

Comment:
Well. Actually `rolling.apply` allow to pass `args` and `kwargs` to function.It seems doesn't work right now in numba. @PokhodenkoSA  has details.

But please, at least specify this limitation in docs

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.window.Rolling.apply.html